Formula/ship.rb

```diff
@@ -0,0 +1,28 @@
+class Ship < Formula
+  desc "Reducing the Overhead of Maintaining 3rd-party Applications in Kubernetes"
+  homepage "https://www.replicated.com/ship"
+  url "https://github.com/replicatedhq/ship/archive/v0.29.0.tar.gz"
+  sha256 "5a82fde0ecec241f404536573937e8123a79f40aa6ca78c79b8d89f75a5e78d0"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+  depends_on "yarn" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    srcpath = buildpath/"src/github.com/replicatedhq/ship"
+    srcpath.install buildpath.children
+    srcpath.cd do
+      system "make", "VERSION=#{version}", "build-minimal"
+      bin.install "bin/ship"
```

Comment:
No doc, no man page?

Reply:
No, we don't currently have a man page for ship.  Ship docs and description are available at www.replicated.com/ship and the README in the code repo.